packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,66 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have the following properties/attributes in addition to the ones provided by the FAST anchor:
+    -   `appearance`:
+        -   `"text"`: (Default) the "standalone" version in the design doc. Looks like plain text, but gets and underline on hover.
+        -   `"inline-text"`: like `"text"`, but always shows an underline.
```

Comment:
Does `inline` have a known meaning of "underline"? I'm just wondering why we don't call this `underline`.

Reply:
I had originally considered using an attribute called `underline`, but Jesse made a good point that going with `inline` would be more conceptual, and would let us change things more easily if designers changed what an inline link should look like. I have since switched from `inline` being a separate attribute to being one value of `appearance`.

Maybe a better answer to your question, though, is that for inline anchors, we really need to do more than just give it a perma-underline. We'll need to change the height and display mode to ensure it aligns/sizes properly within text content.